functests/utils/nodes/nodes.go

```diff
@@ -32,6 +33,21 @@ func GetByRole(c client.Client, role string) ([]corev1.Node, error) {
 	return nodes.Items, nil
 }
 
+// GetNonCNFNode retuns node with no worker-rt label
+func GetNonCNFNode(c client.Client, role string) ([]corev1.Node, error) {
```

Comment:
- `c` is unused and can be removed (actually it can be removed in all func here and replaced with direct use of `testclient.Client` like you do here as well)
- IMHO we can even remove `role` and use a hardcoded const for "worker" like we do for `testutils.RoleWorkerRT`
- we use workerRT (at least as const name), and return multiple worker nodes, so maybe rename the func `GetNonRTWorkers()`?